.github/ISSUE_TEMPLATE/Proposal.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Proposal
+about: Marked doesn't do this thing and I think it should
+ 
+---
+ 
+**What pain point are you perceiving?.**
```

Comment:
This one is where having the marked version made the least sense - but that was the limitation of having to put all three in one.

Reply:
I'm confused. There is no version requested for the Proposal template.